app/api/video_stream_moderators.py

```diff
@@ -0,0 +1,56 @@
+from flask_rest_jsonapi import ResourceDetail, ResourceList, ResourceRelationship
+
+from app.api.bootstrap import api
+from app.api.helpers.query import event_query
+from app.api.schema.video_stream_moderators import VideoStreamModeratorSchema
+from app.models import db
+from app.models.video_stream_moderator import VideoStreamModerator
+
+
+class VideoStreamModeratorList(ResourceList):
+    """
+    List and create video_stream_moderators
+    """
+
+    def query(self, view_kwargs):
+        query_ = self.session.query(VideoStreamModerator)
+        query_ = event_query(query_, view_kwargs)
+
+        return query_
+
+    view_kwargs = True
+    decorators = (
+        api.has_permission(
+            'is_coorganizer', fetch='video_stream_id', model=VideoStreamModerator
```

Comment:
How can you get if a user is coorganizer from `video_stream_id`. You need event ID for that